website/sidebars.json

```diff
@@ -0,0 +1,210 @@
+{
+  "docs": {
+    "Getting started": [
+      "design/index",
+      "tutorials/index",
+      "operations/single-server",
+      "tutorials/cluster"
+    ],
+    "Tutorials": [
+      "tutorials/tutorial-batch",
+      "tutorials/tutorial-kafka",
+      "tutorials/tutorial-batch-hadoop",
+      "tutorials/tutorial-query",
+      "tutorials/tutorial-rollup",
+      "tutorials/tutorial-retention",
+      "tutorials/tutorial-update-data",
+      "tutorials/tutorial-compaction",
+      "tutorials/tutorial-delete-data",
+      "tutorials/tutorial-ingestion-spec",
+      "tutorials/tutorial-transform-spec",
+      "tutorials/tutorial-kerberos-hadoop"
+    ],
+    "Design": [
+      "design/architecture",
+      "design/segments",
+      "design/processes",
+      "dependencies/deep-storage",
+      "dependencies/metadata-storage",
+      "dependencies/zookeeper"
+    ],
+    "Data ingestion": [
+      "ingestion/index",
+      "ingestion/data-formats",
+      "ingestion/schema-design",
+      "ingestion/data-management",
+      {
+        "type": "subcategory",
+        "label": "Stream ingestion",
+        "ids": [
+          "development/extensions-core/kafka-ingestion",
+          "development/extensions-core/kinesis-ingestion",
+          "ingestion/tranquility",
+          "ingestion/standalone-realtime"
+        ]
+      },
+      {
+        "type": "subcategory",
+        "label": "Batch ingestion",
+        "ids": [
+          "ingestion/native-batch",
+          "ingestion/hadoop"
+        ]
+      },
+      "ingestion/tasks",
+      "ingestion/faq"
+    ],
+    "Querying": [
+      "querying/sql",
+      {
+        "type": "subcategory",
+        "label": "Native query types",
+        "ids": [
+          "querying/querying",
+          "querying/timeseriesquery",
+          "querying/topnquery",
+          "querying/groupbyquery",
+          "querying/scan-query",
+          "querying/timeboundaryquery",
+          "querying/segmentmetadataquery",
+          "querying/datasourcemetadataquery",
+          "querying/searchquery",
+          "querying/select-query"
+        ]
+      },
+      "querying/multi-value-dimensions",
+      "querying/lookups",
+      "querying/joins",
+      "querying/multitenancy",
+      "querying/caching",
+      "development/geo"
+    ],
+    "Configuration": [
+      "configuration/index",
+      "development/extensions",
```

Comment:
Very nice outline! Also, linking straight to .md is great.

One thing though: I see that the list of extensions (extensions.md) has been moved under Configuration. I think it's better to mention in the release note this change and any other structural change, to let existing users know and adjust.

Reply:
Ah yes, I did move it there, since I thought it made more sense there. (The Extensions page is written in a user-facing style, and so I don't think it belonged in the Development section.)